### PR TITLE
feat: support sts client

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

when `config.sts === true`, it will create STSClient

Closes https://github.com/eggjs/egg/issues/1476

### CODE REVIEW

File: lib/oss.js [hunk 1]
```diff
@@ -13,12 +14,16 @@ function checkBucketConfig(config) {
 
 module.exports = app => {
   app.addSingleton('oss', (config, app) => {
-    config = Object.assign({}, config, { urllib: app.urllib });
+    config = Object.assign({}, config, { urllib: app.httpclient });
```
Comment (popomore) [inline lib/oss.js]: bug

**Comment by popomore** [comment]
@dead-horse 过了

**Comment by popomore** [comment]
@dead-horse 看看

**Comment by fengmk2** [comment]
1.1.0

**Pull request merged**